apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/AbstractLocalStackTest.java

```diff
@@ -12,19 +12,17 @@ public abstract class AbstractLocalStackTest {
   /** The system property defined in pom.xml containing the image to use for test AWS stack. */
   public static final String TEST_CONTAINER_AWS_IMAGE_PROPERTY = "its.testcontainer.aws.image";
 
-  /** The default test container image to use when nothing is provided. */
-  public static final String TEST_CONTAINER_AWS_IMAGE_DEFAULT = "localstack/localstack:2.2.0";
-
   /** Global container used by multiple tests. Will be destroyed automatically by test container. */
   protected static final LocalStackContainer localstack;
 
   // Creates the instance when first test class loads.   Starts all services that we use in BFD.
   // Add more here if we need any others.
   static {
-    var localStackImageName =
-        System.getProperty(TEST_CONTAINER_AWS_IMAGE_PROPERTY, TEST_CONTAINER_AWS_IMAGE_DEFAULT);
+    var localStackImageName = System.getProperty(TEST_CONTAINER_AWS_IMAGE_PROPERTY);
```

Comment:
`TEST_CONTAINER_AWS_IMAGE_DEFAULT` was outdated, unused. Removed so that we don't continue to forget maintaining this version when the poms are updated.